mlops/parallelm/mlops/mlops_exception.py

```diff
@@ -1,12 +1,69 @@
+import functools
+import logging
+
 class MLOpsException(Exception):
     """
     Exception that will be returned by the MLOps class on error
     """
     pass
 
+class MLOpsExceptionHandler(object):
+    handlers = {}
```

Comment:
Aren't these attributes supposed to be private - prefixed with leading '_'

Reply:
done